tailwind.config.ts

```diff
@@ -72,6 +72,27 @@ function dashedBordersPlugin({ matchUtilities, theme }: PluginAPI) {
   );
 }
 
+function dashedBordersAccentPlugin({ matchUtilities, theme }: PluginAPI) {
```

Comment:
Why was this needed?

Reply:
It was for the dashed border color for the Fill in the option
<img width="218" alt="image" src="https://github.com/user-attachments/assets/daac37a4-4419-4203-953b-b2163a9839bd">